Groupon UP/Groupon UP/BrowseConstants.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
Should this be `DealDetailsConstants`? (since you renamed the `BrowseView` to `DealDetailsView`)